src/query/executor/engine.go

```diff
@@ -168,5 +152,5 @@ func (e *Engine) ExecuteExpr(ctx context.Context, parser parser.Parser, opts *En
 
 // Close kills all running queries and prevents new queries from being attached.
 func (e *Engine) Close() error {
-	return e.tracker.Close()
+	return nil
```

Comment:
If we're not actually closing anything at the moment, should we get rid of `Close`, or add a TODO?

Reply:
might still need close to flush some metrics/traces in the future. Will leave it here for now. I'll update the comment